libs/langchain-azure-cosmosdb/src/tests/caches.int.test.ts

```diff
@@ -31,33 +31,42 @@ function vectorEmbeddingPolicy(
         path: "/embedding",
         dataType: "float32",
         distanceFunction,
-        dimensions: 0,
+        dimensions: 505,
```

Comment:
Would be best to pass embeddings as input of the `vectorEmbeddingPolicy` function, and generate a test embedding to get the length. This way the tests can be run with any embedding model